### PR TITLE
[Edit] Python Added Codebyte in .lower()

### LINKED ISSUE #2981: [Edit] Python Add Codebyte
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add a Codebyte to the listed entry.
https://www.codecademy.com/resources/docs/python/strings/lower

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ChiragAgg5k**
### Description

Edited lower.md to add a codebyte. Closes  #2981 .

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/lower/lower.md [hunk 1]
```diff
@@ -67,3 +67,17 @@ if my_string.lower() == "amazing!":
...
 # Output: "Isn't that just AMAZING!""
 ```
+
+## Codebyte Example
+
+Comparing `my_favorite_color` to `user_favorite_color` using `.lower()`:
+
+```codebyte/python
+my_favorite_color = "Blue"
+user_favorite_color = input("What is your favorite color? : ")
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/lower/lower.md:77]: the input() function isn't supported by the Codecademy compiler so this example will throw an error. Please revise the example. Additionally, note that docs code uses 2 spaces, not tabs. The use of tabs will return file errors.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@ChiragAgg5k I've added a couple comments. Please update, and tag me when you've finished making changes. Thanks

**Comment by ChiragAgg5k** [comment]
@caupolicandiaz I have made the changes.

**Comment by ChiragAgg5k** [comment]
Hey @caupolicandiaz hope you are not too busy. Just mentioning you again in case this pr slipped past you.

**Review by SSwiniarski — APPROVED** [review]
@ChiragAgg5k Looks good.

**Pull request merged**